src/core/cosmosdb_containers.tf

```diff
@@ -0,0 +1,29 @@
+#
+# LolliPOP containers
+#
+resource "azurerm_cosmosdb_sql_container" "lollipop_pubkeys" {
+
+  name                = "lollipop-pubkeys"
+  resource_group_name = data.azurerm_cosmosdb_account.cosmos_api.resource_group_name
+  account_name        = data.azurerm_cosmosdb_account.cosmos_api.name
+  database_name       = data.azurerm_cosmosdb_sql_database.db.name
+
+  partition_key_path    = "/assertionRef"
+  partition_key_version = 2
+
+  dynamic "autoscale_settings" {
+    for_each = [""]
+    content {
+      max_throughput = 1000
+    }
+  }
+
+  lifecycle {
+    ignore_changes = [
+      autoscale_settings
+    ]
+  }
+
+  default_ttl = -1
```

Comment:
@BurnedMarshal @gquadrati @fabriziopapi @AleDore @Garma00 do we want a default TTL of 15 minutes here?

Reply:
IMHO we must manage ttl in an applicative way to avoid changing this value on infra in the future

Reply:
I agree with @AleDore